src/peft/tuners/lokr/config.py

```diff
@@ -103,12 +106,12 @@ class LoKrConfig(LycorisConfig):
         default=None,
         metadata={"help": "List of module names or regex expression of the module names to exclude from LoKr."},
     )
-    init_weights: bool = field(
+    init_weights: Union[bool, str] = field(
```

Comment:
```suggestion
    init_weights: Union[bool, Literal["lycoris"]] = field(
```

a bit more precise type annotation.